ext/opcache/ZendAccelerator.c

```diff
@@ -1180,7 +1180,7 @@ int validate_timestamp_and_record(zend_persistent_script *persistent_script, zen
 	}
 }
 
-int validate_timestamp_and_record_ex(zend_persistent_script *persistent_script, zend_file_handle *file_handle)
+zend_result validate_timestamp_and_record_ex(zend_persistent_script *persistent_script, zend_file_handle *file_handle)
 {
 	int ret;
```

Comment:
Might as well change this too -> zend_result.

Reply:
Aye!